src/index.html

```diff
@@ -11,7 +11,215 @@
+        <legend>Registration</legend>
+
+        <label class="form_field" for="email">E-mail:</label>
+        <input
+          class="form_element"
+          type="email"
+          name="email"
+          id="email"
+          placeholder="email@example.com"
+          minlength="6"
+          maxlength="30"
+          required
+          multiple
+        >
+        <br>
+
+        <label class="form_field" for="password">Password:</label>
+        <input
+          type="password"
+          name="password"
+          id="password"
+          minlength="6"
+          maxlength="20"
+          required
+        >
+
+      </fieldset>
+
+      <fieldset class="fieldSet">
+        <legend>An interesting fact about you!</legend>
+
+        <div>
+          <label class="form_field form_element" for="cats">Do you love cats?</label>
+          <label for="yes">
+            <input
+              class="form_element"
+              type="radio"
+              name="cats"
+              id="yes"
+              checked
+            >
+            Yes
+          </label>
+          <label class="form_field" for="no">
+            <input
+              class="form_element"
+              type="radio"
+              name="cats"
+              id="no"
+            >
+            No
+          </label>
+
+        </div>
+
+        <div>
+          <label class="form_field" for="favColor">What is your favorite color?</label>
+          <input
+            class="form_element"
+            type="color"
+            name="favColor"
+            id="favColor"
+            value="black"
+          >
+
+        </div>
+
+        <div>
+          <label class="form_field" for="timeBed">What time do you go to bed?</label>
+          <input
+            class="form_element"
+            type="time"
+            name="timeBed"
+            id="timeBed"
+            step="2"
+          >
+          <br>
+        </div>
+
+        <div>
+          <label class="form_field form_element" for="favCar">What are your favorite brand of cars?</label>
+          <select
+            name="favCar"
+            id="favCar"
+            multiple
+          >
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+          <br>
+        </div>
+
+        <div>
+          <label class="form_field" for="rateWork">How do you rate our work?</label>
+          <input
+            type="range"
+            name="rateWork"
+            id="rateWork"
+            value="0"
+          >
+        </div>
+
+      </fieldset>
+
+      <fieldset class="fieldSet">
+        <legend>Additional info</legend>
+
+        <div>
+          <label class="form_field" for="comments">Comments</label>
+          <textarea
+            class="form_element"
+            name="comments"
+            id="comments"
+            autocomplete="off">
+          </textarea>
```

Comment:
```suggestion
            autocomplete="off"
          ></textarea>
```
![image](https://user-images.githubusercontent.com/96409563/216153589-164095eb-224b-4ed7-9e03-80a3d5f37a5b.png)

![image](https://user-images.githubusercontent.com/96409563/216153466-a4edb27c-ca53-43dc-8b5b-ca29d2c831e1.png)